lib/jekyll/commands/serve/servlet.rb

```diff
@@ -21,10 +21,36 @@ def initialize(server, root, callbacks)
         # up with a different preference on which comes first.
 
         def search_file(req, res, basename)
-          # /file.* > /file/index.html > /file.html
+          # /file.* -> /file.html
```

Comment:
Ah, in that case I'll clarify and move it above `search_file` since it applies to both.